messages/german.json

```diff
@@ -0,0 +1,1387 @@
+        "harlot": ["Dirne", "Dirnen"],
+        "hunter": ["Jäger", "Jägern"],
+        "insomniac": ["Schlafloser", "Schlaflose"],
+        "investigator": ["Ermittler", "Ermittler"],
+        "jester": ["Hofnarr", "Hofnarr"],
+        "lycan": ["Lykaner", "Lykaner"],
+        "mad scientist": ["Verrückter Wissenschaftler", "Verrückte Wissenschaftler"],
+        "matchmaker": ["Armor", "Armor"],
+        "mayor": ["Bürgermeister", "Bürgermeister"],
+        "minion": ["Untergebener", "Untergebene"],
+        "monster": ["Monster", "Monster"],
+        "mystic": ["Mystiker", "Mystiker"],
+        "oracle": ["Orakel", "Orakel"],
+        "piper": ["Rattenfänger", "Rattenfänger"],
+        "priest": ["Priester", "Priester"],
+        "prophet": ["Prophet", "Propheten"],
+        "seer": ["Seher", "Seher"],
+        "shaman": ["Schamane", "Schamanen"],
+        "sharpshooter": ["Scharfschütze", "Scharfschützen"],
+        "sorcerer": ["Hexer", "Hexer"],
+        "succubus": ["Sukkubus", "Sukkuben"],
+        "time lord": ["Zeitherr", "Zeitherren"],
+        "tough wolf": ["Harter Wolf", "Harte Wölfe"],
+        "turncoat": ["Überläufer", "Überläufer"],
+        "traitor": ["Verräter", "Verräter"],
+        "vengeful ghost": ["Rachsüchtiger Geist", "Rachsüchtige Geister"],
+        "vigilante": ["Mitglieder der Bürgerwehr", "Mitglieder der Bürgerwehr"],
+        "villager": ["Bürger", "Bürger"],
+        "village drunk": ["Saufbold", "Saufbolde"],
+        "warlock": ["Zauberer", "Zauberer"],
+        "werecrow": ["Werkrähe", "Werkrähe"],
+        "werekitten": ["Werkätzchen", "Werkatze"],
+        "wild child": ["Wildes Kind", "Wilde Kinder"],
+        "wolf": ["Wolf", "Wölfe"],
+        "wolf cub": ["Wolfsjunge", "Wolfsjungen"],
+        "wolf gunner": ["Wolfsschütze", "Wolfsschützen"],
+        "wolf mystic": ["Wolfmystiker", "Wolfsmystiker"],
+        "wolf shaman": ["Wolfschamane", "Wolfschamanen"],
+        "**": "The following items are names for the 'team reveal' role reveal option",
+        "village member": ["Dorfmitglied", "Dorfmitglieder"],
+        "neutral player": ["Neutrale-Spieler", "Neutraler-Spieler"],
+        "wolfteam player": ["Wolfteam-Spieler", "Wolfteam-Spieler"],
+        "***": "The following items are special keys used in player stats",
+        "lover": ["Liebhaber", "Liebhaber"],
+        "vg activated": ["rg aktiviert", "rgs aktiviert"],
+        "vg driven off": ["rg deaktiviert", "rg deaktiviert"],
+        "entranced": ["verzaubert", "verzaubert"],
+        "****": "The following items are old roles that are no longer in use, but need to be here for legacy reasons",
+        "bureaucrat": ["Bürokrat", "Bürokraten"]
+    },
+    "_gamemodes": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Gamemodes",
+        "aleatoire": "aleatoire",
+        "alpha": "Alpha",
+        "boreal": "Nördlich",
+        "charming": "charmant",
```

Comment:
The point of charming is, that is has a piper, who charms people. So "verzaubernd" or something close to it would be more fitting.